packages/web-components/src/global/_variables.scss

```diff
@@ -51,6 +51,7 @@ $font-size-h3: 2rem;
 $font-size-h4: 1.7rem;
 $font-size-h5: 1.5rem;
 $font-size-h6: 1.5rem;
+$v3-font-base-size: 16px;
```

Comment:
@Andrew565 Just a head's up that this `_variables.scss` file is generated from the [css-library](https://github.com/department-of-veterans-affairs/component-library/tree/main/packages/css-library) so we can't edit it directly (it'll just get overwritten). But I added it there so it should generate when we run css-library build.

I have renamed it to `$v3-font-base-size` to match the pattern we have used for other v3 specific variables.

Reply:
Here is the variable added to the css-library: https://github.com/department-of-veterans-affairs/component-library/blob/84049d7c06890ca10f4f4d4a67543effb85c8783/packages/css-library/tokens/fonts.json#L30-L33